examples/functions/deckPlans/DeckPlans-Example_bus.xml

```diff
@@ -0,0 +1,1897 @@
+@ 09A 09B +     + 09C 09D @
+@:::::::::+     +:::::::::@
+@ 10A 10B   10E   10C 10D @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@  Upper Back   @@@@@@
+
+     DECK PLAN (Doouble decker bus)
+         DECK (lower)           
+            OTHERSPACE (Driver)
+            PASSENGER SPACE Upper Seating
+              DECK ENTRANCEs (Front, Rear, Stairwell)
+                ENTRANCE EQUIPMENT
+                ENTRANCE SENSOR
+              PASSENGER SPOTs (x25)
+ 
+			  SPOT ROWs (x8)
+			  SPot COlumns (x5)
+            OTHER SPACE (Stairwell)
+        DECK (upper)                
+            PASSENGER SPACE
+               DECK ENTRANCE ( Stairwell)
+               PASSENGER SPOTs (x41)
+                 SEAT EQUIPMENT Bench
+                 SPOT SENSOR  
+			  SPOT ROWs (x10)
+			  SPot COlumns (x5)
+-->
+	<!-- Deck plan assignment to a service
+  The Deck plan is shown assigned to a specific vehicle on a specifc journey
+   The journey is split into three parts . The bus runs with its doors open in the central part in the city
+    and with its door s shut  in the
+
+-->
+	<PublicationTimestamp>2020-12-17T09:30:47.0Z</PublicationTimestamp>
+	<ParticipantRef>SYS001</ParticipantRef>
+	<!-- ======WHAT WAS REQUESTED ========== -->
+	<PublicationRequest version="1.0">
+		<RequestTimestamp>2020-12-17T09:30:46.0Z</RequestTimestamp>
+		<ParticipantRef>SYS002</ParticipantRef>
+		<Description>Request for Erebus 1 tariff</Description>
+		<topics>
+			<NetworkFrameTopic>
+				<selectionValidityConditions>
+					<AvailabilityCondition version="any" id="r1">
+						<FromDate>2020-01-01T00:00:00Z</FromDate>
+					</AvailabilityCondition>
+				</selectionValidityConditions>
+				<NetworkFilterByValue>
+					<objectReferences>
+						<OperatorRef version="any" ref="noc:OP-erebus"/>
+					</objectReferences>
+				</NetworkFilterByValue>
+			</NetworkFrameTopic>
+		</topics>
+	</PublicationRequest>
+	<PublicationRefreshInterval>P1M</PublicationRefreshInterval>
+	<Description>Example of DeckPlans</Description>
+	<!-- =============== RESULTS =========== -->
+	<dataObjects>
+		<CompositeFrame version="1.0" id="mb:CF-deck_plan_example_bus" dataSourceRef="mb:DS-erebus" responsibilitySetRef="mb:RS-infra">
+			<ValidBetween>
+				<FromDate>2023-01-01T00:00:00</FromDate>
+				<ToDate>2023-12-31T12:00:00</ToDate>
+			</ValidBetween>
+			<Name>Deck Plan  Example for DOuble DEcker Bus </Name>
+			<Description>This is an example showing how one might encode a Deck plan for a double decker bus in NeTEx. It includes seat  an, entrance and sensor equipment.</Description>
+			<!--==== CODESPACEs ==== -->
+			<codespaces>
+				<Codespace id="CS-mb_data">
+					<Xmlns>mb</Xmlns>
+					<XmlnsUrl>http://www.erebus.eu/</XmlnsUrl>
+					<Description>erebus data</Description>
+				</Codespace>
+			</codespaces>
+			<!--==== FRAME DEFAULTS ==== -->
+			<FrameDefaults>
+				<!---  The default means that actually we could leave off all id s that belong to the mb: namespace. -->
```

Comment:
Should comment be included?

Reply:
this comment repeats itself in several places in the examples. I wont mark all of them.